requirements.txt

```diff
@@ -0,0 +1,12 @@
+PyYAML==6.0.1
+ruamel.yaml==0.18.6
+numpy==1.26.4
+pandas==2.2.2
+scipy==1.13.0
+matplotlib==3.9.0
+torch==2.2.2
+tqdm==4.66.4
+hmmlearn==0.3.2
+opencv-python-headless==4.9.0.80
+umap-learn==0.5.6
+h5py==3.11.0
```

Comment:
it would probably be better to use `>=` for some of these @vinicvaz 
we don't want to make it more strict than necessary

Reply:
How would we define which one to use `>=`? Or should I put all with `>=`?